packages/launcher-app/.env.openshift

```diff
@@ -1,11 +1,9 @@
 REACT_APP_CREATOR_API_URL=/launch/creator
 REACT_APP_LAUNCHER_API_URL=/launch/api
 
-REACT_APP_AUTHENTICATION=openshift
-REACT_APP_OPENSHIFT_CLIENT_ID=
+REACT_APP_AUTHENTICATION=oauth-openshift
+REACT_APP_OAUTH_OPENSHIFT_CLIENT_ID=
+REACT_APP_OAUTH_OPENSHIFT_URL=
 
-REACT_APP_OPENSHIFT_AUTH_URL=
-
-REACT_APP_GITHUB_CLIENT_ID=
-REACT_APP_GITHUB_SECRET=
-REACT_APP_GITHUB_VALIDATE_URI=https://cors-anywhere.herokuapp.com/https://github.com/login/oauth/access_token
```

Comment:
don't we want to keep this for local testing?

Reply:
We do I put it back ;-)